api/v1beta2/cloudstackmachine_types.go

```diff
@@ -135,10 +136,12 @@ type CloudStackMachineStatus struct {
 
 	// Status indicates the status of the provider resource.
 	// +optional
+	// +k8s:conversion-gen=false
 	Status *string `json:"status,omitempty"`
 
 	// Reason indicates the reason of status failure
 	// +optional
+	// +k8s:conversion-gen=false
```

Comment:
can these 3 still be converted between v2 and v3?

Reply:
I'm prepping another PR for conversions. That will handle all conversions properly, including this.